packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -1055,6 +1055,41 @@ class SliverFillViewport extends SliverMultiBoxAdaptorWidget {
   /// the viewport in the main axis.
   final double viewportFraction;
 
+  /// The delegate that provides the children for this widget.
```

Comment:
Maybe use a macro instead of repeating this text from SliverMultiBoxAdaptorWidget?